x-pack/plugins/observability_solution/infra/public/pages/metrics/page_template.tsx

```diff
@@ -48,6 +51,7 @@ export const MetricsPageTemplate: React.FC<LazyObservabilityPageTemplateProps> =
           beats: {
             title: noMetricIndicesPromptPrimaryActionTitle,
             description: noMetricIndicesPromptDescription,
+            href: onboardingLocator?.useUrl({ category: 'infra' }),
```

Comment:
Same applies here as per https://github.com/elastic/kibana/pull/184477/files#r1622102394

Reply:
Addressed in https://github.com/elastic/kibana/pull/184477/commits/409a64c2cf124c13a1d5524343c0bc5c603c0cb4.